src/App.js

```diff
@@ -15,20 +15,40 @@ chance.mixin({
 });
 
 const allFriends = Array.from({length: 20}, () => chance.friend());
+const bestFriends = allFriends.filter(friend => friend.isBestFriend);
 
 console.log(allFriends);
 
 export default class App extends Component {
   render() {
     return (
       <div className="App">
-
-        {/*Make a list of all your friends*/}
-
-        {/*Make a list of just your best friends*/}
-
-        {/*Show the average age of your best friends*/}
-
+          <div className="friendList">
+              {/*Make a list of all your friends*/}
+              <h1>My Friends</h1>
+              <ul>
```

Comment:
Thanks for the suggestion! I've created a new `FriendList` component to display both of the lists.